static/src/js/components/ErrorPageNotFound/__tests__/ErrorPageNotFound.test.jsx

```diff
@@ -0,0 +1,46 @@
+import {
+  MemoryRouter,
+  Route,
+  Routes
+} from 'react-router-dom'
+import { render, screen } from '@testing-library/react'
+import React from 'react'
+
+import AuthContext from '@/js/context/AuthContext'
+
+import ErrorPageNotFound from '../ErrorPageNotFound'
+
+vi.mock('@/js/utils/errorLogger')
+vi.mock('uuid', () => ({
```

Comment:
After you make the `crypto.randomUUID()` update, this mock is not needed. During test setup we are mocking crypto like this:
```
vi.mock('crypto', () => ({
  default: {
    randomUUID: () => 'mock-uuid'
  }
}))
```